lib/model/sdp/protocol.py

```diff
@@ -97,9 +97,12 @@ def _close(self):
         self._connection.close()
         self._is_connected = False
 
-    def _send(self, data_dict):
+    def _send(self, data_dict, **kwargs):
         self.logger.debug(f'{self.__class__.__name__} _send called with {data_dict}')
-        return self._connection.send(data_dict)
+        return self._connection.send(data_dict, **kwargs)
+
+    def _check_command(self, command, value):
```

Comment:
Wenn das in connection schon definiert ist, braucht es hier nicht noch mal identisch zu stehen. Idealerweise sollte es reichen, wenn es nur hier steht (s. connection.py)